Cargo.toml

```diff
@@ -21,12 +21,12 @@ errno = "0.2.8"
 libc = "0.2.105"
 
 [build-dependencies]
-bindgen = { version = "0.60.1", default-features = false, features = ["runtime"] }
+bindgen = { version = "0.59.2", default-features = false, features = ["runtime"] }
```

Comment:
Is there a reason to downgrade this version?

Reply:
Ah yah - forgot to mention this. The `cross` Docker images are based on Ubuntu 18.04. The update to 20 is ongoing but not yet released. Bindgen 0.60 requires a more recent version of `libclang` that is not available in 18.04 and upgrading llvm is hard. The pragmatic approach to tackle this is to downgrade `bindgen` because the [changes](https://github.com/rust-lang/rust-bindgen/blob/master/CHANGELOG.md) in `bindgen` between `0.59.2` and `0.60.1` are not relevant here.

As an alternative a differnent base images could be used. e.g from Circle CI.